packages/typespec-client-generator-core/generated-defs/Azure.ClientGenerator.Core.Legacy.ts

```diff
@@ -0,0 +1,46 @@
+import type { DecoratorContext, Model } from "@typespec/compiler";
+
+/**
+ * Adds support for client-level multiple levels of inheritance.
+ *
+ * This decorator will update the models returned from TCGC to include the multi-level inheritance information.
```

Comment:
Can we add more details om what scenario this is enabling? What does multi-level inheritance mean in this context. It might confuse users, for example multi-level inheritance is techincally possible in TypeSpec

```tsp
model A {
  id: string;
}

model B extends A{
  name: string;
}

model C extends B{
  birthday: utcDateTime;
}
```